src/display/annotation_storage.js

```diff
@@ -208,12 +213,15 @@ class AnnotationStorage {
  * contents. (Necessary since printing is triggered synchronously in browsers.)
  */
 class PrintAnnotationStorage extends AnnotationStorage {
-  #serializable = null;
+  #serializable;
 
-  constructor(parent) {
+  constructor({ serializable }) {
     super();
     // Create a *copy* of the data, since Objects are passed by reference in JS.
-    this.#serializable = structuredClone(parent.serializable);
+    const map = structuredClone(serializable.map);
```

Comment:
We should avoid the image cloning here, maybe in using the transfer parameter in`structuredClone`:
https://developer.mozilla.org/en-US/docs/Web/API/structuredClone#parameters

Reply:
Yes, I think that'll work. (Unfortunately it does look a little ugly since the `structuredClone` polyfill doesn't support transfers.)